.gitignore

```diff
@@ -11,6 +11,7 @@ Pipfile*
 db.sqlite3
 media/
 static/
+!transport_nantes/*/static/
```

Comment:
Why is this correct?

Rather than excluding what you don't want and hoping you didn't miss anything, please write a single regex that matches what you do want.

Reply:
static/ bans every static folder (including js and css files)
!transport_nantes/*/static/ adds an exception (with `!`)  for static folders inside our apps. 
As you know, transport_nantes/ folder is the base directory, inside it we have all our apps, and most have a subdirectory named static. 

What's wrong with this, and how is a regex better than the .gitignore syntax ?